src/Core/Transformers/Message.ts

```diff
@@ -24,8 +24,9 @@ export default class MessageTransformer {
       client,
       message,
       command,
+      arg: args.length > 0 ? args[0] : "",
       args,
-      members: client.guilds.get(process.env.GUILD_ID!)!.members,
+      members: {} as any /* client.guilds.get(process.env.GUILD_ID!)!.members */,
```

Comment:
## Blocking

Why? If this field isn't going to be used anymore, you should remove it and fix all the conflicts